core/src/main/java/org/mskcc/cbio/oncokb/util/AlterationUtils.java

```diff
@@ -324,7 +324,10 @@ public static void annotateAlteration(Alteration alteration, String proteinChang
                     String groupSix = m.group(6);
                     String groupSixWithoutDigits = MainUtils.removeDigits(groupSix);
 
-                    if (groupSixWithoutDigits.length() != groupSix.length() && groupSixWithoutDigits.length() > 0) {
+                    if (groupSixWithoutDigits.contains("*")) {
```

Comment:
If we adjust the regex in the comment on L303, then we can continue using contains. Otherwise currently, it would make more sense to use endsWith.

Reply:
Answered in standup